src/main/java/duke/ui/Ui.java

```diff
@@ -0,0 +1,111 @@
+package duke.ui;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import duke.task.Task;
+/**
+ * Represents the user interface class that
+ * deals with interactions with the user.
+ *
+ * @author Leong Jia Hao Daniel
+ */
+public class Ui {
```

Comment:
I like that there are multiple ways that your code displays the output based on the user's input